src/bundle/Resources/public/css/variables.css

```diff
@@ -0,0 +1,37 @@
+:root {
+    --ez-base-font-family: "Noto Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
"Noto Sans" is not really a standard font (yet), I put what we used currently in PlatformUI 1.x as a fallback, any opinion on that @SylvainGuittard @inakijv ?